pages/index.html

```diff
@@ -131,6 +135,26 @@ <h2>Tabs</h2>
     </div>
   </tabs-panel-sk>
 
+  <h2>select-sk and multi-select-sk</h2>
+
+  <select-sk>
+    <div>select</div>
+    <div>any</div>
+    <div selected>one</div>
+  </select-sk>
+  <br>
```

Comment:
Use CSS to make select-sk and multi-select-sk display: block and then drop the <br>.

Reply:
They are display:block by default, I just wanted a bit of space between them.  CSS spacing added instead.